orbit/pkg/keystore/keystore_darwin.go

```diff
@@ -0,0 +1,158 @@
+//go:build darwin && cgo
```

Comment:
Should we add a package level lock? (to protect the methods from being executed concurrently)

Reply:
Wouldn't hurt, as protection for future.